docs/api/dialog.md

```diff
@@ -23,37 +23,38 @@ console.log(dialog)
 
 The `dialog` module has the following methods:
 
-### `dialog.showOpenDialog([browserWindow, ]options[, callback])`
-
-* `browserWindow` [BrowserWindow](browser-window.md) (optional)
-* `options` Object
-  * `title` String (optional)
-  * `defaultPath` String (optional)
-  * `buttonLabel` String (optional) - Custom label for the confirmation button, when
-    left empty the default label will be used.
-  * `filters` [FileFilter[]](structures/file-filter.md) (optional)
-  * `properties` String[] (optional) - Contains which features the dialog should
-    use. The following values are supported:
-    * `openFile` - Allow files to be selected.
-    * `openDirectory` - Allow directories to be selected.
-    * `multiSelections` - Allow multiple paths to be selected.
-    * `showHiddenFiles` - Show hidden files in dialog.
-    * `createDirectory` _macOS_ - Allow creating new directories from dialog.
-    * `promptToCreate` _Windows_ - Prompt for creation if the file path entered
-      in the dialog does not exist. This does not actually create the file at
-      the path but allows non-existent paths to be returned that should be
-      created by the application.
-    * `noResolveAliases` _macOS_ - Disable the automatic alias (symlink) path
-      resolution. Selected aliases will now return the alias path instead of
-      their target path.
-    * `treatPackageAsDirectory` _macOS_ - Treat packages, such as `.app` folders,
-      as a directory instead of a file.
-  * `message` String (optional) _macOS_ - Message to display above input
-    boxes.
-  * `securityScopedBookmarks` Boolean (optional) _masOS_ _mas_ - Create [security scoped bookmarks](https://developer.apple.com/library/content/documentation/Security/Conceptual/AppSandboxDesignGuide/AppSandboxInDepth/AppSandboxInDepth.html#//apple_ref/doc/uid/TP40011183-CH3-SW16) when packaged for the Mac App Store.
-* `callback` Function (optional)
-  * `filePaths` String[] - An array of file paths chosen by the user
-  * `bookmarks` String[] _macOS_ _mas_ - An array matching the `filePaths` array of base64 encoded strings which contains security scoped bookmark data. `securityScopedBookmarks` must be enabled for this to be populated.
+### `dialog.showOpenDialog(opts)`
+
+* `opts` Object
+  * `browserWindow` [BrowserWindow](browser-window.md) (optional)
+  * `options` Object (optional)
```

Comment:
It's really weird to have nested options-objects. These should just be at the top-level with `browserWindow`. There's nothing special about the `browserWindows` compared to `title`. They're all just options.